lib/infrastructure/theme/app_colors.dart

```diff
@@ -2,7 +2,8 @@ import 'package:flutter/material.dart';
 
 class AppColors {
   static const white = Colors.white;
-  static const background = Color.fromRGBO(15, 76, 131, 1);
-  static const accent = Color.fromRGBO(250, 255, 0, 1);
-  static const backgroundBottom = Color.fromRGBO(3, 43, 80, 1);
+  static const grey = Colors.grey;
```

Comment:
remove this color
you should add only some specific color in this class, which not exist in Material colors class